tests/test_hooks/test_ema_hook.py

```diff
@@ -79,6 +81,10 @@ def setUp(self):
         self.temp_dir = tempfile.TemporaryDirectory()
 
     def tearDown(self):
+        # `FileHandler` should be closed in Windows, otherwise we cannot
+        # delete the temporary directory
+        logging.shutdown()
+        MMLogger._instance_dict.clear()
```

Comment:
Do we have a more elegant way to shut down those file handlers in the future?

Reply:
We'll provide a `TestCase` to create and delete global variance.